packages/synapse-react-client/src/components/SynapseSearchPageResults/HasAccessChip.tsx

```diff
@@ -0,0 +1,51 @@
+import { RestrictionUiType } from '../HasAccess/AccessIcon'
+import { useGetRestrictionUiType, HasAccessV2 } from '../HasAccess/HasAccessV2'
+import { Chip } from '@mui/material'
+import { searchResultsCardChipStyles } from '../SynapseSearchPageResults/chipStyles'
+
+export interface HasAccessChipProps {
+  entityId: string
+  size?: 'normal' | 'small'
+}
+
+export function HasAccessChip({ entityId }: HasAccessChipProps) {
+  const restrictionUiType = useGetRestrictionUiType(entityId, {
+    enabled: false,
+  })
+
+  const getAccessText = () => {
```

Comment:
Move this out of the component and pass `restrictionUiType` as a parameter

Reply:
done